{"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ValueError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"symbolKind":"case","title":"PythonError.ValueError(_:)","role":"symbol","roleHeading":"Case","externalID":"s:7SwiftPy11PythonErrorO05ValueD0yACSScACmF","modules":[{"name":"SwiftPy"}]},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ValueError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"sections":[],"schemaVersion":{"minor":3,"major":0,"patch":0},"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"]]},"kind":"symbol","variants":[{"paths":["\/documentation\/swiftpy\/pythonerror\/valueerror(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/ValueError(_:)"},"references":{"doc://SwiftPy/documentation/SwiftPy":{"role":"collection","url":"\/documentation\/swiftpy","abstract":[{"text":"A fast and lightweight Python interpreter with Swift macro binding tools.","type":"text"}],"type":"topic","title":"SwiftPy","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","kind":"symbol"},"doc://SwiftPy/documentation/SwiftPy/PythonError/ValueError(_:)":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ValueError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"abstract":[],"title":"PythonError.ValueError(_:)","role":"symbol","type":"topic","url":"\/documentation\/swiftpy\/pythonerror\/valueerror(_:)","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/ValueError(_:)","kind":"symbol"},"doc://SwiftPy/documentation/SwiftPy/PythonError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"PythonError","kind":"identifier"}],"kind":"symbol","abstract":[],"navigatorTitle":[{"text":"PythonError","kind":"identifier"}],"type":"topic","url":"\/documentation\/swiftpy\/pythonerror","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError","role":"symbol","title":"PythonError"}}}